network.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * This file is part of the libvirt-go-xml project
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * Copyright (C) 2017 Lian Duan <blazeblue@gmail.com>
+ *
+ */
+
+package libvirtxml
+
+import (
+	"encoding/xml"
+)
+
+type NetworkBridge struct {
+	Name            string `xml:"name,attr,omitempty"`
+	STP             string `xml:"stp,attr,omitempty"`
+	Delay           string `xml:"delay,attr,omitempty"`
```

Comment:
@chemhack /cc @berrange Not, you can have a *int, where there nil pointer is empty, and if it's not then it points to the value. This is what we use in mgmt for example. This works as expected with the YAML parser, I assume it would also work with the XML one.